src/helper/remote_config/listener.hpp

```diff
@@ -28,8 +27,7 @@ class product_listener_base {
     virtual void on_unapply(const config &config) = 0;
 
 protected:
-    std::shared_ptr<remote_config::remote_config_service>
-        _remote_config_service;
+    std::shared_ptr<service_config> _remote_config_service;
```

Comment:
Similarly to my previous comment, it's not really a `remove_config_service` and also please stop using `_` in front of identifiers, use it after.